polypie.py

```diff
@@ -32,9 +31,7 @@ def from_parameter(parameter):
         )
 
 
-def _call_func(func_key, args=None, kwargs=None):
-    args = args or ()
-    kwargs = kwargs or {}
+def _call_func(func_key, args=(), kwargs={}):
```

Comment:
i heard of that, but i don't get it. each time this function gets called, a new object will be created. so, what is worrying about it?